pillarbox-cast/src/main/java/ch/srgssr/pillarbox/cast/PillarboxCastPlayer.kt

```diff
@@ -42,16 +71,17 @@ import com.google.android.gms.cast.framework.media.RemoteMediaClient
  * @param seekBackIncrementMs The [seekBack] increment, in milliseconds.
  * @param seekForwardIncrementMs The [seekForward] increment, in milliseconds.
  * @param maxSeekToPreviousPositionMs The maximum position for which [seekToPrevious] seeks to the previous [MediaItem], in milliseconds.
- * @param castPlayer The underlying [CastPlayer] instance to which method calls will be forwarded.
  * @param trackSelector The [CastTrackSelector] to use when selecting tracks from [TrackSelectionParameters].
+ * @param castPlayer The underlying [CastPlayer] instance to which method calls will be forwarded.
  */
-class PillarboxCastPlayer(
+class PillarboxCastPlayer internal constructor(
     private val castContext: CastContext,
     context: Context? = null,
     mediaItemConverter: MediaItemConverter = DefaultMediaItemConverter(),
```

Comment:
Why not remove all default values?

Reply:
Totally we can remove the default values.